scenes/quests/story_quests/stella/1_stella_stealth/stella_stealth_components/stella_firekeeper.dialogue

```diff
@@ -1,8 +1,8 @@
 # SPDX-FileCopyrightText: The Threadbare Authors
 # SPDX-License-Identifier: MPL-2.0
 ~ start
-{{player_name}}:“Do you know why Orion is missing a star?”
-{{npc_name}}:"Who is Orion? A hunter?" the Firekeeper chuckles. "No, child. Those three stars are the Hearthstones of Creation. The missing one? That was our fire.”
-“The Maize God once kindled the cosmic flame. Without it, the world is out of balance.”
-“But fire is never freely given. It must be taken back.”
+{{player_name}}: Do you know why Orion is missing a star?
+{{npc_name}}: Who is Orion? A hunter? the Firekeeper chuckles. No, child. Those three stars are the Hearthstones of Creation. The missing one? That was our fire.”
```

Comment:
Did you mean to leave “the Firekeeper chuckles” here, or leave the unpaired trailing quote?? I would write:

```suggestion
{{npc_name}}: Who is Orion? A hunter? Ha! [wait=0.5] No, child. Those three stars are the Hearthstones of Creation. The missing one? That was our fire.
```